src/newLaunch/seed/stages/stage4.ts

```diff
@@ -158,9 +158,9 @@ export class Stage4 extends BaseStage<ISeedConfig> {
       (this.numberService.fromString(fromWei(this.launchConfig.tokenDetails.maxSupply, this.launchConfig.tokenDetails.projectTokenInfo.decimals)) *
         this.launchConfig.launchDetails.pricePerToken)) {
       message = "Funding Maximum cannot be greater than Maximum Project Token Supply times the Project Token Exchange Ratio";
-    } else if (!this.launchConfig.launchDetails.vestingPeriod || !(this.launchConfig.launchDetails.vestingPeriod >= 0)) {
+    } else if (this.launchConfig.launchDetails.vestingPeriod===null || this.launchConfig.launchDetails.vestingPeriod===undefined) {
```

Comment:
Is there any chance can be undefined?  Maybe easier to check if is a number?  Like `Number.isInteger` and >= 0?  (note that you are allowed < 0)